docs/api/api-spec.md

```diff
@@ -142,18 +142,41 @@ Context channels allows a set of apps to share a stateful piece of data between
 
 There are two types of channels, which are functionally identical, but have different visibility and discoverability semantics.  
 
-1. The 'system' ones, which have a well understood identity. One is called 'default'.
+1. The 'system' ones, which have a well understood identity. One is called 'global'.
 2. The 'app' ones, which have a transient identity and need to be revealed
-
-The 'system' channels include a 'default' channel which serves as the backwards compatible layer with the 'send/broadcast context' above. There are some reserved channel names for future use. Currently this is just 'global'.
+ 
 
 ### Joining Channels
-Apps can join channels.  An app can only be joined to one channel at a time.  When an app joins a channel it will automatically recieve the current context for that channel, except if the channel joined is the 'default'.
+Apps can join channels.  An app can only be joined to one channel at a time.  When an app joins a channel it will automatically recieve the current context for that channel.
 
-When an app is joined to a channel, calls to fdc3.broadcast and listeners added through fdc3.addContextListener will be routed to that channel.  If an app is not explicitly joined to a channel, it is on the 'default' channel.  It is up to the desktop agent to determine the behavior of the 'default' channel, and if context is automaticaly broadcast over it or not.
+When an app is joined to a channel, calls to fdc3.broadcast and listeners added through fdc3.addContextListener will be routed to that channel.  If an app is not joined to a channel these methods will be no-ops, but apps can still choose to listen and broadcast to specific channels via the methods on the `Channel` class.  
 
 It is possible that a call to join a channel could be rejected.  If for example, the desktop agent wanted to implement controls around what data apps can access.  
 
+Joining channels in FDC3 is intended to be a behavior initiated by the end user. For example: by color linking or apps being grouped in the same workspace.  Most of the time, it is expected that apps will be joined to a channel by mechanisms outside of the app.  Always, there SHOULD be a clear UX indicator of what channel an app is joined to. 
+
+### The 'global' Channel
+The 'system' channels include a 'global' channel which serves as the backwards compatible layer with the 'send/broadcast context' behavior in FDC3 1.0.  A desktop agent MAY choose to make membership in the 'global' channel the default state for apps on start up.  
+
+The 'global' channel should be returned as part of the response from the `fdc3.getSystemChannels` call.  Desktop Agents may want to filter out the 'global' option in their UI for system channel pickers.
+
+
+#### Examples
+
+An app gets the current context of the `global` as a fallback if no other context is set.
```

Comment:
I am not sure what this means? Can you expand on what you mean by a fallback?

Reply:
some platforms may have a tiered approach, where there is a global context (maybe coming from a top-level search) and more localized contexts created with channels.  In that case, if an app doesn't have a context coming from a specific channel it is joined to it may want to pull the global context.

Reply:
I am tempted to say this is slightly inconsistent with the rest of the behaviour, and therefore introduces a bit of confusion. I think channels should always work the same way, regardless of what context it is. If you are on no channel, no context will be recorded. If you want context to be recorded, join a channel.

In this particular case I would prefer if the spec doesn't mention this behaviour. Someone can still do this if they like, as the standard wouldn't explicitly forbade it, but I don't personally think it makes much sense.

Reply:
I agree with @rikoe here. I feel we are kind of reintroducing a new special behavior for "global" with this wording.

The way I expect - If I join `global` channel and also add a explicit `channel.contextListener` for `red` channel, I am expecting to get messages from both. Your statement seems to suggest you will only receive `red` channel messages (not sure if that is what you meant) and I will fallback to global when I remove the listener for red channel.

Reply:
@rikoe , @vmehtafds - some clarifications:

1. in the proposal, the "global" channel would NOT have any special behavior, it would work just like any other channel.  The only thing special about it, would be that it is a reserved system channel name so that: platform owners could choose to start apps on global by default, app developers could choose to start their apps on global by default, app developers could choose to listen to and broadcast to the global channel, etc.  These would all be opt-in behaviors.
2. in the code example, yes, if you joined `red` and added a contextListener on `global`, you would get messages for both.   I am not dictating only using global as fallback - that was just an example of what I think is a probable use-case (I could be wrong)

@rikoe I am not clear on the concerns (I don't see how this is going against channels always working the same) but I am fine changing this example to something else.  Do you have any suggestions?

Reply:
@nkolba I see what you mean now - adding an explicit listener to global. No other concerns with what you describe.

However, `An app gets the current context of the `global` as a fallback if no other context is set.` is confusing.

The language here indicates special fallback behaviour without having to set up extra explicit listeners, which is unexpected.

If you are joined to global, getCurrentContext should get you the context on global, if not, you will get nothing. I think it will be clearer if we just remove the word "fallback" everywhere.

Of course you can still directly retrieve the channel itself and listen/retrieve the context on it, but that is a separate workflow from the join channel one.

Reply:
@rikoe . Yes, if you are joined to `global`, then `fdc3.getCurrentContext` only gets you context from global - nothing in the example contradicts that.   The example is attempting to show *both* using `fdc3.joinChannel` and directly retrieving channels and setting explicit listeners. I'll try to make the language and the example clearer.

Reply:
Is `joinChannel` async? 🤔 I would argue the same applies. I do agree that both join and leave should both be consistent though...

Arguably both join and leaving channels should be instant no-ops, but happy to go with the majority view here.

Reply:
Change 
`An app gets the current context of the `global` as a fallback if no other context is set.` 
to
`An app can retrieve the context on the `global` channel by accessing it directly, regardless if it is currently "joined" to the global channel or not.`
Or something like that...

If you want to make it even more clear:
```ts
// retrieve current fdc3 context
const context = await fdc3.getCurrentContext("fdc3.instrument")
// context is null, as not currently joined to a channel

const globalChannel = await fdc3.getSystemChannels.filter(c => c.id === "global")
const globalContext = await fdc3.getCurrentContext("fdc3.instrument")
// context is instrument AAPL on the global channel

fdc3.joinChannel('global')
const context = await fdc3.getCurrentContext('fdc3.instrument')
// top-level context is now instrument AAPL as well because we have joined the global channel
```

Reply:
@rikoe just pushed some language cleanup

Reply:
@rikoe updated example pushed